README.md

```diff
@@ -31,6 +31,38 @@ npm start
 
 This will start a local server running on http://localhost:8080/.
 
+# Test
+
+If you need to run all the tests (unit and contract) just run
+
+```bash
+npm test
+```
+
+## Unit
+
+You can run the unit tests a single time with
+
+```bash
+npm run test:unit
+```
+
+or keep them running if you are modifying them and want to check the results live (TDD FTW!):
+
+```bash
+npm run test:unit:watch
+```
+
+## Contract
+
+To generate the Pact contract files and upload them to you local Pact Broker, you have to run
+
+```bash
+npm run test:contract
+```
+
+and be sure to have [Pact Broker](https://github.com/reevoo/pact_broker) running.
+
```

Comment:
We now need to run ```bundle install``` as part of the test setup.
Can we add that in the here as well?